src/main/java/com/minecraftabnormals/endergetic/core/events/CompatEvents.java

```diff
@@ -6,23 +6,31 @@
 import com.minecraftabnormals.endergetic.core.EndergeticExpansion;
 
 import net.minecraft.entity.LivingEntity;
+import net.minecraft.potion.Effect;
 import net.minecraft.util.ResourceLocation;
 import net.minecraftforge.event.entity.living.PotionEvent;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 import net.minecraftforge.fml.common.Mod;
 import net.minecraftforge.registries.ForgeRegistries;
 
+import java.util.Objects;
+
 /**
- * @author tessdotcpp
+ * @author abigailfails
  * Events for compatibility with the Savage & Ravage mod.
  */
 @Mod.EventBusSubscriber(modid = EndergeticExpansion.MOD_ID)
 public final class CompatEvents {
+
+	private static final ResourceLocation SHRINKING_ID = new ResourceLocation("savageandravage:shrinking");
+	private static final ResourceLocation GROWING_ID = new ResourceLocation("savageandravage:growing");
+
 	@SubscribeEvent
 	public static void onPotionExpire(PotionEvent.PotionExpiryEvent event) {
 		LivingEntity affected = event.getEntityLiving();
-		boolean isBabyEffect = event.getPotionEffect().getPotion() == ForgeRegistries.POTIONS.getValue(new ResourceLocation("savageandravage:shrinking"));
-		if (isBabyEffect || event.getPotionEffect().getPotion() == ForgeRegistries.POTIONS.getValue(new ResourceLocation("savageandravage:growth"))) {
+		Effect effectType = Objects.requireNonNull(event.getPotionEffect()).getPotion();
+		boolean isBabyEffect = effectType == ForgeRegistries.POTIONS.getValue(SHRINKING_ID);
+		if (isBabyEffect || effectType == ForgeRegistries.POTIONS.getValue(GROWING_ID)) {
```

Comment:
There are many blank lines above here that aren't needed.

Reply:
Not sure which blank lines you're referring to. Did you mean the one between the GROWING_ID declaration and @SubscribeEvent?

Reply:
I think GitHub was just displaying it weirdly.

Reply:
Ah, i see.